docs/ga4gh-pedigree.rst

```diff
@@ -0,0 +1,92 @@
+GA4GH Pedigree defines a conceptual model of individuals and their relationships to allow other standards, such as
+Phenopackets to be able to create an implementation of this model using native concepts. For example, the Phenopacket
+schema representation of an individual in the GA4GH Pedigree is a ref:`rstphenopacket`.
+
+The GA4GH Pedigree Standard allows for much greater expressivity of the relationships between individuals compared to the
+PED model used in the ref:`rstfamily` message due to defining the relationships using the `Kinship ontology (KIN) <http://purl.org/ga4gh/kin.owl>`_
+
+
+Data model
+##########
+
+The data model described here is transcribed from the `documentation <https://pedigree.readthedocs.io/en/latest/pedigree-model.html#pedigree>`_
+
+Pedigree
+========
+
+.. csv-table::
+   :header: Field, Type, Multiplicity, Description
+
+   id, string, 1..1, External identifier for the family being investigated
+   narrative, string, 0..1, Summary of the pedigree resource for human interpretation
+   date, string, 0..1, The date the pedigree was collected or last updated, as ISO full or partial date, i.e. YYYY, YYYY-MM, or YYYY-MM-DD
+   reason, string, 0..1, The reason for pedigree collection, a health condition of focus being investigated in the family; if any Individual has the affected property defined, it refers to this condition.
+   status, string, 0..1, Status of the pedigree resource collection
+   index_patients, string, 0..*, Identified Individual(s) in the family of a health condition of focus being investigated: Proband, Consultand, First Person Tested Positive
+   individuals, ref:`rstphenopacket`, 0..*, Collection of Individual who are the members of this pedigree
+   relationships, ref:`rsga4ghrelationship`, 0..*, Collection of Relationship between the individuals who are the members of this pedigree
+
+
+.. _rsga4ghrelationship:
+
+Relationship
+============
+
+.. csv-table::
+   :header: Field, Type, Multiplicity, Description
+
+   individual, string, 1..1, Identifier of the subject Individual; equivalent to the Biolink "subject"
+   relation, ref:`rstontologyclass`, 1..1, The relationship the individual has to the relative (e.g., if the individual is the relative's biological mother, then relation could be isBiologicalMotherOf [KIN:027]); terms should come from the `KIN Ontology <http://purl.org/ga4gh/kin.owl>`_.
+   relative, string, 1..1, Identifier of the relative Individual; equivalent to the Biolink "object"
+
+
+The identifier values used in the ``individual`` and ``relative`` fields **must** match a value in the **id** element
+used in the :ref:`rstindividual` ``subject`` of the phenopackets listed in the ``pedigree`` ``individuals`` field.
+
+
+Example
+#######
+
+In this example we have simplified the phenopackets to include only the minimum required information to make a pedigree.
+Ordinarily there should be more fields (e.g. the metadata) within the Phenopacket.
```

Comment:
The name of the class should be capitalized, MetaData, I think?

Reply:
Also, I think we wanted to write GA4GH Phenopacket Schema as a proper noun but not to capitalize "a" phenopacket.